frontend/src/app/offers/offer-detail/offer-detail.component.ts

```diff
@@ -4,26 +4,43 @@ import { Component, OnInit } from '@angular/core';
 import { Observable } from 'rxjs/Observable';
 
 import { environment } from '../../../environments/environment';
-import { Offer } from '../../homepage-offer/offers.model';
+import { ApiOffer } from '../../homepage-offer/offers.model';
 import { OffersService } from '../../homepage-offer/offers.service';
+import { User } from 'app/user';
+import { AuthService } from 'app/auth.service';
+import { Organization } from 'app/organization/organization.model';
+import { combineLatest } from 'rxjs/operators/combineLatest';
 
 @Component({
   selector: 'volontulo-offer-detail',
   templateUrl: './offer-detail.component.html',
   styleUrls: ['./offer-detail.component.scss'],
 })
 export class OfferDetailComponent implements OnInit {
-  public offer$: Observable<Offer>;
+  public offer$: Observable<ApiOffer>;
   public djangoRoot = environment.djangoRoot;
```

Comment:
It seems that we are not using djangoRoot anywhere here or in a template. If it is true, then please delete it and also delete the environment import if not needed anymore.